webrtc/src/main.c

```diff
@@ -17,9 +17,9 @@
 #include <re_dbg.h>
 
 
-static const char *modpath = "/usr/local/lib/baresip/modules";
+static const char *modpath = "/usr/local/share/baresip";
```

Comment:
I noticed that the modules path has changed for cmake install. is this correct ?

Reply:
Yes I think we should fix this: https://github.com/baresip/baresip/pull/2068